core/services/workflows/syncer/workflow_registry.go

```diff
@@ -353,7 +380,7 @@ func (w *workflowRegistry) readRegistryEvents(ctx context.Context, don capabilit
 			}
 
 			for _, event := range events {
-				err := w.handler.Handle(ctx, event.Event)
+				err := w.handler.HandleEvent(ctx, event.Event)
```

Comment:
_nit_ consider renaming to like `w.events.Handle` or `w.handle.Event`

Reply:
Reverted name. Everything is handled as events now.